gbpservice/neutron/services/servicechain/plugins/ncp/node_drivers/nfp_node_driver.py

```diff
@@ -273,6 +274,16 @@ def get_plumbing_info(self, context):
 
         if service_type in [pconst.FIREWALL, pconst.VPN]:
             plumbing_request['plumbing_type'] = 'gateway'
+            # plumber will return stitching network PT instead of consumer
+            # as chain is instantiated while creating provider group.
+            if (self._check_for_fw_vpn_sharing(context, service_type) and
+                    service_type == pconst.VPN):
+                # For fw, vpn sharing, request pts only once.
+                # Request pts only for firewall, if both fw, vpn is in chain,
+                # and current service type is vpn, dont request pts.
+                LOG.info(_("Not requesting plumber for PTs for service type "
+                           "%s") % service_type)
```

Comment:
In case of FW+FW sharing, interfaces are not shared but only service VM? Is this correct?